src/main/java/no/nav/pto/veilarbportefolje/util/DbUtils.java

```diff
@@ -100,24 +99,27 @@ public static OppfolgingsBruker mapTilOppfolgingsBruker(ResultSet rs, UnleashSer
                 .setEgen_ansatt(parseJaNei(rs.getString("sperret_ansatt"), "sperret_ansatt"))
                 .setEr_doed(parseJaNei(rs.getString("er_doed"), "er_doed"))
                 .setDoed_fra_dato(toIsoUTC(rs.getTimestamp("doed_fra_dato")))
-                .setVeileder_id(rs.getString("veilederident"))
                 .setFodselsdag_i_mnd(Integer.parseInt(FodselsnummerUtils.lagFodselsdagIMnd(rs.getString("fodselsnr"))))
                 .setFodselsdato(FodselsnummerUtils.lagFodselsdato(rs.getString("fodselsnr")))
                 .setKjonn(FodselsnummerUtils.lagKjonn(rs.getString("fodselsnr")))
-                .setOppfolging(parseJaNei(rs.getString("OPPFOLGING"), "OPPFOLGING"))
-                .setNy_for_veileder(parseJaNei(rs.getString("NY_FOR_VEILEDER"), "NY_FOR_VEILEDER"))
-                .setNy_for_enhet(isNyForEnhet(rs.getString("veilederident")))
                 .setTrenger_vurdering(OppfolgingUtils.trengerVurdering(formidlingsgruppekode, kvalifiseringsgruppekode))
                 .setVenterpasvarfrabruker(toIsoUTC(rs.getTimestamp("venterpasvarfrabruker")))
                 .setVenterpasvarfranav(toIsoUTC(rs.getTimestamp("venterpasvarfranav")))
-                .setManuell_bruker(identifiserManuellEllerKRRBruker(rs.getString("RESERVERTIKRR"), rs.getString("MANUELL")))
                 .setEr_sykmeldt_med_arbeidsgiver(OppfolgingUtils.erSykmeldtMedArbeidsgiver(formidlingsgruppekode, kvalifiseringsgruppekode))
                 .setVedtak_status(Optional.ofNullable(vedtakstatus).map(KafkaVedtakStatusEndring.VedtakStatusEndring::valueOf).map(KafkaVedtakStatusEndring::vedtakStatusTilTekst).orElse(null))
                 .setVedtak_status_endret(toIsoUTC(rs.getTimestamp("VEDTAK_STATUS_ENDRET_TIDSPUNKT")))
                 .setAnsvarlig_veileder_for_vedtak(ansvarligVeilederForVedtak)
                 .setTrenger_revurdering(OppfolgingUtils.trengerRevurderingVedtakstotte(formidlingsgruppekode, kvalifiseringsgruppekode, vedtakstatus))
                 .setHar_delt_cv(parseJaNei(rs.getString(HAR_DELT_CV), HAR_DELT_CV))
-                .setCv_eksistere(parseJaNei(rs.getString(CV_EKSISTERE), CV_EKSISTERE));
+                .setCv_eksistere(parseJaNei(rs.getString(CV_EKSISTERE), CV_EKSISTERE))
+                .setOppfolging(parseJaNei(rs.getString("OPPFOLGING"), "OPPFOLGING")); // Oppfolging hentes fra Oracle helt til at alt er migrert
+        if (!brukAvOppfolgingsdataPaPostgres(unleashService)) {
```

Comment:
how is this connected with src/main/java/no/nav/pto/veilarbportefolje/postgres/opensearch/PostgresOpensearchMapper.java and opposite condition there..

Reply:
this method is retrieving data from oracle, the other one from postgres